pkg/util/winutil/users_test.go

```diff
@@ -0,0 +1,29 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018-present Datadog, Inc.
+
+// +build windows
+
+package winutil
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetSidFromUser(t *testing.T) {
+	sid, err := GetSidFromUser()
```

Comment:
Some assertions around the value/type of `sid` would be good too otherwise we're just testing the `err` return value.

Reply:
Yes, but there's only one code path that returns a non-nil error value.

Reply:
ditto: I can assert `NotNil`.